packages/ui-components/src/styles/markdown.css

```diff
@@ -134,11 +134,10 @@ main {
       text-sm
       dark:border-neutral-800;
 
+    /* Common border and text styles */
     th,
     td {
-      @apply max-xs:block
-        max-xs:border-l-0
-        border
+      @apply border
         border-r-0
         border-t-0
         border-neutral-200
```

Comment:
```suggestion
        border-neutral-200
        whitespace-nowrap
```
I think this will fix the issue of words wrapping in the cells. WDYT?

Reply:
Could you share a screenshot of the issue you’re referring to?

Reply:
![Screenshot_20250830-220900_Chrome](https://github.com/user-attachments/assets/e42142e8-bf32-4795-9659-a9d7083b627f)